client/rpc-api/src/chain/mod.rs

```diff
@@ -54,6 +55,23 @@ pub trait ChainApi<Number, Hash, Header, SignedBlock> {
 	#[rpc(name = "chain_getFinalizedHead", alias("chain_getFinalisedHead"))]
 	fn finalized_head(&self) -> Result<Hash>;
 
+	/// Set offchain storage under given key and prefix.
+	#[rpc(name = "chain_setOffchainStorage")]
```

Comment:
```suggestion
	#[rpc(name = "offchain_localStorageSet")]
```

I'd suggest to keep the naming from internal modules, i.e. `Local Storage` and verb at the end.
Also IMHO it deserves a separate namespace in the RPC (i.e. not `chain_`, but rather `offchain_`) and a separate file for this.